google/datalab/utils/commands/_utils.py

```diff
@@ -336,6 +338,62 @@ def parse_config(config, env, as_dict=True):
   return config
 
 
+def parse_config_for_selected_keys(content, env, keys):
```

Comment:
Will do that in a separate change. For now it is partially covered in CommandParser tests.